lab-davide/gulpfile.js

```diff
@@ -0,0 +1,57 @@
+const gulp = require('gulp');
+const mocha = require('gulp-mocha');
+const eslint = require('gulp-eslint');
+
+const filepath = ['*.js', 'lab-davide/lib/*.js', 'lab-davide/test/*.js'];
+
+gulp.task('default', function() {
```

Comment:
make a separate test task, then have your default task call it. People shouldn't have to read the code of your tasks to figure out what they do--the name should make it clear immediately.